<launch>
    <node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="apriltag_ros_continuous_node" clear_params="true" >
        <!-- Remap topics from those used in code to those on the ROS network -->
        <remap from="image_rect" to="/mechmind/color_image" />
        <remap from="camera_info" to="/mechmind/camera_info" />
        <param name="camera_frame" type="str" value="mechmind_camera" />
        <param name="publish_tag_detections_image" type="bool" value="false" />      <!-- default: false -->
        <rosparam command="load" file="$(find tams_head_mesh)/config/settings.yaml" ns="$(arg node_namespace)" />
        <rosparam command="load" file="$(find tams_head_mesh)/config/tags.yaml" ns="$(arg node_namespace)" />
    </node>
</launch>
